src/Scripting/VM/VirtualMachine.hpp

```
@@ -0,0 +1,395 @@
+	/* Tests whether a bit in a bitmask is flipped to 1 (True), or 0 (False). */
+	FORCE_INLINE bool testBit(const Compiler::RawBitmaskT bitmask, const int bit) const;
+
+	/* Converts a value to its string representation. */
+	std::string variantToString(const AsTL::StackValue &val) const;
+};
+
+
+template<typename T>
+inline FORCE_INLINE T VirtualMachine::popFromVMStackAs(Compiler::OperandType opType) {
+	AsTL::IDX consumed{};
+	const T v = castFromStack<T>(opType, &consumed);
```

Clang (14.0.6)

[warning] variable name 'v' is too short, expected at least 3 characters

(readability-identifier-length)